about.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/about.css">
+  <script src="https://kit.fontawesome.com/1ada02b4f0.js" crossorigin="anonymous"></script>
+  <title>About the tourism </title>
+</head>
+
+<body>
+  <header>
+    <div class="top-navbar">
+      <ul>
+        <li>
+          <a href="#"><i class="fa fa-twitter"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa fa-facebook"></i></a>
+        </li>
+        <li><a href="#">Celebrities</a></li>
+        <li><a href="#">Vlog</a></li>
+        <li><a href="#"> Logout</a></li>
+      </ul>
+    </div>
+    <div class="main-navbar flex-center">
+      <div class="logo">
+        <a href="index.html"><img src="./assets/business-logo.jpg" alt="fashion logo" /></a>
+      </div>
+      <nav class="navbar">
+        <div class="hamburger">
+          <img src="./assets/hamburger.svg" alt="Menu Hamburger" />
+        </div>
+        <ul class="nav-links">
+          <li class="close-menu-btn">
+            <button type="submit">
+              <span>&times;</span>
+            </button>
+          </li>
+          <li><a href="index.html" id="current">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#">Models</a></li>
+          <li><a href="#">Tickets</a></li>
+          <li id="leehar-line-lg" class="flex-center d-none d-block-lg"><a href="#">CC Campaign</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
```

Comment:
- [ ] I know you have worked up to now but you still have to improve the design of your `navbar` by the following
buy following the same design of the template. check the difference down here 👇  👇 

your design | 
------------| 
![image](https://user-images.githubusercontent.com/65068771/147107628-bc67ac65-947d-4a08-b4c0-de0800acf589.png) | 
The template | 
![image](https://user-images.githubusercontent.com/65068771/147107861-8ce91b2e-273d-4239-8485-c0d16501e19a.png) |

Reply:
Thanks 👍 I just used the Logo I can. We are allowed to use any Logo as per the requirement.

Reply:
You can check out the new pull request I've modified all the `navbar and navbar-link` to suit the template. 😎😎